src/main/kotlin/fi/metatavu/timebank/api/controllers/UsersController.kt

```diff
@@ -0,0 +1,42 @@
+package fi.metatavu.timebank.api.controllers
+
+import fi.metatavu.timebank.api.impl.translate.UserTranslator
+import fi.metatavu.timebank.api.keycloak.KeycloakController
+import fi.metatavu.timebank.model.User
+import org.keycloak.admin.client.resource.UsersResource
+import org.keycloak.representations.idm.UserRepresentation
+import java.util.UUID
+import javax.enterprise.context.ApplicationScoped
+import javax.inject.Inject
+
+/**
+ * Controller for User objects
+ */
+@ApplicationScoped
+class UsersController {
+
+    @Inject
+    lateinit var keycloakController: KeycloakController
+
+    @Inject
+    lateinit var userTranslator: UserTranslator
+
+    /**
+     * Fetches User from keycloak
+     *
+     * @param userId UUID
+     * @return UserRepresentation
+     */
+    fun findUser(userId: UUID): UserRepresentation? {
+        return keycloakController.findUserById(userId)
+    }
+
+    /**
+     * Lists Users found in Keycloak
+     *
+     * @return List<User>
+     */
```

Comment:
```suggestion
    /**
     * Lists Users found in Keycloak
     *
     * @return List of UserRepresentations
     */
```

Reply:
JavaDoc doesn't need types, the JavaDoc tool itself parses the types from code. IMO my suggestion is more human readable.